src/App.css

```diff
@@ -22,3 +22,20 @@
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
 }
+
+.coffee-list {
+  margin:10px;
+  padding: 10px;
+  width: 150px;
+  border: 2px solid gray;
+  background-color: #e3e3e3;
+  float: left;
```

Comment:
we'll eventually use flexbox and not deal with floats, but for now it is kool and the gang